src/base/preferences.cpp

```diff
@@ -718,6 +718,16 @@ void Preferences::useSystemIconTheme(bool enabled)
 }
 #endif
 
+bool Preferences::useSystemFileIcons() const
```

Comment:
Frankly speaking, `Preference` deprecation seems to be a bit premature. I think we should either at least teach "Advanced" settings page to accept settings from outside or agree an interface for configurable components. Because right now I have to uncover an implementation detail just to make it accessible for  options dialog. IMO, our current approach is best suited for applications that use context menus for configurations, but not the central options dialog. I will make requested changes, through.